.changeset/curvy-trains-deny.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin": minor
+---
+
+Add componentsProps to Accordion
```

Comment:
Shouldn't this be "Change default styling of accordion"? And does this qualify as a breaking change @jamesricky?

Reply:
I don't think this changeset is needed, as the component was added in this PR.